script/worker

```diff
@@ -118,7 +118,10 @@ sub usage($) {
     }
 }
 
-GetOptions(\%options, "no-cleanup", "instance=i", "isotovideo=s", "host=s", "apikey:s", "apisecret:s", "verbose|v", "help|h",) or usage(1);
+GetOptions(
+    \%options,     "no-cleanup", "instance=i", "isotovideo=s", "host=s", "apikey:s",
+    "apisecret:s", "verbose|v",  "help|h",
```

Comment:
so why is not putting the args into lines on its own here? perltidy seems to be pretty inconsistent